### DISCUSSION TITLE
Autostart GUI farming on reboot

### DISCUSSION THREAD

**Discussion by luotinen** [Chia-Network/chia-blockchain#8640] [Ideas]
Please add an option to resume farming automatically after a reboot. The current procedure requires that the user manually restarts the farming after reboot. If the farming was active when the shutdown or powerless occurred, I think it is safe to assume that the user wants to continue farming after rebooting.

**Comment by hoffmang9** [comment]
Farming already will autostart if you autostart the application. It will default to your first private key - even though it shows the key selection box.

You just need the OS to start the GUI up.

**Comment by BrandtH22** [comment]
@luotinen , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
